frontend/src/fixtures/courseDetailsFixtures.js

```diff
@@ -0,0 +1,41 @@
+const courseDetailsFixtures = {
+  twoCourses: [
+    {
+      quarter: "20211",
```

Comment:
I think we are only looking at data from 2022 onwards (start quarter 20221), so maybe we can look at classes that are from 2022 onwards.